app/models/bus.rb

```diff
@@ -0,0 +1,20 @@
+class Bus < ApplicationRecord
+    validates :name, uniqueness: { case_sensitive: false }
+    validates :name, :typed, :brand, :capacity, presence: true
```

Comment:
For all these attributes can you please check empty string or  `' '` is accepted or not? Thank you.

Reply:
Checked . It returns the expected error.